runtime/queries/markdown/highlights.scm

```diff
@@ -71,17 +104,17 @@
 (thematic_break) @punctuation.special
 
 (task_list_marker_unchecked) @markup.list.unchecked
+
 (task_list_marker_checked) @markup.list.checked
 
-((block_quote) @markup.quote (#set! "priority" 90))
+((block_quote) @markup.quote
+  (#set! "priority" 90))
 
 [
   (block_continuation)
   (block_quote_marker)
 ] @punctuation.special
 
-[
-  (backslash_escape)
-] @string.escape
+[ (backslash_escape) ] @string.escape
```

Comment:
Your choice whether to keep or strip these brackets.

Reply:
I'm keeping it. Can always look back at it later anyways